src/jurisdictions/PA/dictionary.txt

```diff
@@ -256,6 +256,7 @@ CandidateContest	Senator in the General Assembly;6th Senatorial District	Pennsyl
 CandidateContest	Senator in the General Assembly;8th Senatorial District	Pennsylvania;General Assembly Senate District 8
 CandidateContest	United States Senator;Statewide	Pennsylvania;US Congress;Senate
 ReportingUnit	Pennsylvania;ADAMS County	ADAMS
+ReportingUnit	Pennsylvania;ADAMS County	ADAMS
```

Comment:
@sfsinger19103 I'm still seeing this dupe in here.

Reply:
Should be fixed now. I was looking at ReportingUnit.txt before, not dictionary.txt.